srcpkgs/font-b612/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'font-b612'
 pkgname=font-b612
 version=1.008
-revision=2
+revision=3
 wrksrc="b612-${version}"
 depends="font-util xbps-triggers"
 short_desc="Highly legible font family designed for aircraft cockpit screens"
 maintainer="ghost <gh0st@vivaldi.net>"
-license="EPL-1.0"
-homepage="http://b612-font.com/"
-distfiles="https://github.com/polarsys/b612/archive/${version}.tar.gz"
+license="OFL-1.1"
```

Comment:
should be
```suggestion
license="EPL-2.0, BSD-3-Clause, OFL-1.1"
```
https://github.com/polarsys/b612#license

and add `vlicense edl-v10.html` to `post_install()`